monai/transforms/clinical_preprocessing.py

```
@@ -0,0 +1,70 @@
+    """
+    Preprocess a DICOM series based on modality.
+
+    Args:
+        dicom_path: Path to DICOM file or directory.
+        modality: CT, MR, or MRI.
+
+    Returns:
+        Preprocessed image.
+    """
+    if not isinstance(modality, str):
+        raise TypeError("modality must be a string")
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)